test/test_torch_distributed_multi_all_reduce_xla_backend.py

```diff
@@ -0,0 +1,38 @@
+import os
```

Comment:
Probably not easy in the current test structure, but would be more consistent if we can group all_reduce tests in a single test file using unit test framework. Also, it might be better to have a separate `distributed` test folder under `test/`.

Reply:
yea.. we should eventually have a separate repo for distributed tests